src/debuginfo.cpp

```diff
@@ -68,7 +68,7 @@ struct FuncInfo {
 #else
 struct ObjectInfo {
     const object::ObjectFile* object;
-    size_t size;
+    size_t SectionSize;
```

Comment:
size is supposed to be SymbolSize, not SectionSize. this change could occasional corrupt the backtrace structures.